example/App.js

```diff
@@ -24,6 +24,25 @@ export default class App extends Component {
     Alert.alert(`Package: ${packageSearch}`, `${isInstalled ? 'Installed' : 'Not Installed'}`);
   };
 
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      result: '',
+    };
+  }
+
+  getErrorString(error: any, defaultValue?: string): string {
```

Comment:
Since i updated the example, making him from the scratch i decided to remove flow to become more easier to new user understand and get up running with the library. Would be a good idea remove the flow types according with the other examples in this file?

Reply:
Removed flow definition from the function